src/components/alert/alert.tsx

```diff
@@ -193,10 +202,16 @@ export class Alert {
   //
   //--------------------------------------------------------------------------
 
-  /** Fired when an alert is closed */
+  /* Fired when an alert is going to be closed and before the transition starts. */
+  @Event() calciteAlertBeforeClose: EventEmitter;
```

Comment:
With these added names, I find it hard to determine which is which. In the close instance - does beforeClose mean "after user has requested to close the alert"?

Does "close" mean - has been closed and animation is complete, or the close is about to occur? Maybe we can be more explicit with the naming for those.

Reply:
@macandcheese does this make it more clear?